Makefile

```diff
@@ -975,7 +975,9 @@ else
 	cp bin/gt $(prefix)/bin
 	$(STRIP) $(prefix)/bin/gt
 endif
-	cp -r gtdata $(prefix)/bin
+	test -d $(prefix)/share/genometools \
+	  || mkdir -p $(prefix)/share/genometools
+	cp -r gtdata $(prefix)/share/genometools
```

Comment:
`-a` preserves symlinks and time stamps. May not be relevant here. I do it out of habit.